lms/djangoapps/ccx/models.py

```diff
@@ -24,6 +26,9 @@ class CustomCourseForEdX(models.Model):
     course_id = CourseKeyField(max_length=255, db_index=True)
     display_name = models.CharField(max_length=255)
     coach = models.ForeignKey(User, db_index=True)
+    # if not empty, this field contains a compressed json serialized list of
+    # the master course modules
+    structure_json = CompressedTextField(verbose_name='Structure JSON', blank=True, null=True)
```

Comment:
I am not comfortable with putting a limit: the reason is because there is no limit on [the course structure itself](https://github.com/edx/edx-platform/blob/master/openedx/core/djangoapps/content/course_structures/models.py#L17), so I cannot put an hard limit and still be sure I will never reach it, even if my field will always contain a subset of the actual structure, so this case is pretty unlikely.

What's your reason to enforce a limit?